src/main/java/pl/connectis/spotifyapicli/SpotifyApiCliApplication.java

```diff
@@ -3,12 +3,11 @@
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.builder.SpringApplicationBuilder;
-import org.springframework.boot.context.properties.EnableConfigurationProperties;
-import pl.connectis.spotifyapicli.authorization.AuthorizationConfig;
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
 
 @Slf4j
 @SpringBootApplication
-@EnableConfigurationProperties(AuthorizationConfig.class)
+@ConfigurationPropertiesScan
```

Comment:
chyba można te annotacje pominąć jeśli nie wskazuje sie konkretnego pakietu (tak jak w tym przypadku) ale pewien na 100 % nie jestem

Reply:
Jest raczej potrzebna, jak ją usuwam to się sypią testy